code/dictation.py

```diff
@@ -12,26 +12,31 @@
     desc="Look at surrounding text to improve auto-capitalization/spacing in dictation mode. By default, this works by selecting that text & copying it to the clipboard, so it may be slow or fail in some applications.",
 )
 
-@mod.capture(rule="({user.vocabulary} | <word>)")
+@mod.capture(rule="({user.vocabulary} | <user.abbreviated> | <word>)")
```

Comment:
Mind elaborating what advantage there is to having <user.abbreviated> in the <user.word> capture? My read is this change would just let you say 'word brief difference' and get 'diff' output. You could also just say 'brief difference' and get the same thing.

Reply:
I think the reasoning here is that if there are commands that take `<word>` as an argument, like for instance `proud <word>`, this allows you to use the abbreviated version as the `<word>`. It seems in that case it's worth keeping it.